AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/overlay.js

```diff
@@ -16,8 +16,11 @@ define([
          * @returns {Object}
          */
         getStyles: function (record) {
-            var height = record.styles()['height'].replace('px', '');
-            return {top: (height - 50) + 'px'};
+            var height = record.styles()['height'].replace('px', '') - 50;
+
+            return {
+                   top: height + 'px'
+                };
```

Comment:
Do these changes make the static JS tests fail?

Reply:
seems like static test failed on develop branch, i will try to fix them